python/cugraph-dgl/examples/muti_trainer_MG_example/model.py

```diff
@@ -0,0 +1,145 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# A graphsage GNN model using dgl for node classification
+# with three layers and mean aggregation
+import time
+import dgl
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import torchmetrics.functional as MF
+from dgl.nn import SAGEConv
```

Comment:
Let me switch this to cugraph_dgl.nn
```suggestion
from dgl.nn import SAGEConv
```

Reply:
Done